src/SFA.DAS.ApplyService.Web/Controllers/ApplicationController.cs

```diff
@@ -58,48 +58,44 @@ public async Task<IActionResult> Applications()
 
             if (!applications.Any())
             {
+                //ON-2068 Registered org  with no application created via digital service then
+                //display empty list of application screen
                 if (org != null && org.RoEPAOApproved)
-                {
-                      return await StartApplication(userId);
-                }
+                    return View(applications);
 
-                if (org == null)
-                {
-                    if (await _userService.AssociateOrgFromClaimWithUser())
-                        return await StartApplication(userId);
+                if (org != null)
```

Comment:
Could you tidy this up by doing something like this? 
`                if (org != null)
                {
                    return org.RoEPAOApproved ? return View(applications) : return View("~/Views/Application/Declaration.cshtml");
                }`

Reply:
I have updated the code with:
 if (org != null) return org.RoEPAOApproved ?  View(applications) : View("~/Views/Application/Declaration.cshtml");